### PR TITLE
fix poetry lockfile errors and update installation instruction

### PR DESCRIPTION
**Pull request by ahmubashshir**

### CODE REVIEW

**Comment by m4hi2** [comment]
@ahmubashshir any specific reason behind the restructure?

**Comment by ahmubashshir** [comment]
for easier installation and dependency management.
poetry can install package directly with poetry install.
	Mubashshir

**Comment by m4hi2** [comment]
@ahmubashshir bhai, I see you've imported subprocess but it's not used anywhere. If it's not required please remove it. Also, when I try to run this new branch, I get the following error:

```
python -m routEar
/usr/bin/python: No module named routEar.__main__; 'routEar' is a package and cannot be directly executed
```

File: routEar/app.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+if plt == "win32":
+    from win10toast import ToastNotifier
+    toaster = ToastNotifier()
+elif plt == "linux":
+    from contextlib import closing
+    try:
+        from jeepney import DBusAddress, new_method_call
+        from jeepney.io.blocking import open_dbus_connection
+    except (ImportError, ModuleNotFoundError):
+        import subprocess as sp
```
Comment (m4hi2) [inline routEar/app.py]: Imported but not used.

**Review by m4hi2 — APPROVED** [review]
Thanks for fixing!

**Pull request merged**